sdk/test-utils/recorder/test/utils.spec.ts

```diff
@@ -230,4 +228,28 @@ ultramarine.com/url/PUBLIC
       });
     });
   });
+
+  describe("set environment variables", () => {
+    it("should not fail if the dictionary is empty", () => {
+      env.SECRET = "SECRET";
+      const replaceableVariables = {};
+
+      setEnvironmentVariables(replaceableVariables);
+    });
+
+    it("should succeed if the dictionary has one key-value pair", () => {
+      const replaceableVariables = { SECRET: "FAKE_IT" };
+
+      setEnvironmentVariables(replaceableVariables);
+      expect(env.SECRET).to.equal("FAKE_IT");
```

Comment:
Once setEnvironmentVariables has the environment object parametrized, this unit test won't need to change the global environment object! ✨

Reply:
`setEnvironmentVariables` is supposed to change the global environment object so that the tests in our service sdks like storage or keyvault can use the updated global env variables.

Reply:
That's true. Ok! I only want to minimize what touches the environment variables. If this is the only place, I think that's ok.